src/magento-payments/onboarding-payments.md

```diff
@@ -1,3 +1,129 @@
 ---
 title: Onboarding Payments
----
\ No newline at end of file
+---
+
+Magento Payments is your turnkey self-service solution---including sandbox testing, fraud prevention, and a super simple setup---for providing robust and secure payment processing for your Magento store.
```

Comment:
We are not going to have fraud prevention for EAP, should we clarify it or remove it?

Reply:
Good point, yes.